### PR TITLE
New doc spa

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Resolves

1. 1463

# Explain the feature/fix

1. Updated the content organized in doc spa
2. Changed blog list layout
3. Added updated content for documentation

## Does this PR introduce a breaking change

Yes

## Screenshots

<details>
<summary>View Screenshots</summary>

## Main Page

Screenshot 2022-08-24 at 2 55 37 PM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demoed and the design review approved?

### CODE REVIEW

File: packages/doc-spa/src/pages/index.js [hunk 1]
```diff
@@ -1,51 +1,51 @@
-import React from 'react';
-import clsx from 'clsx';
-import Layout from '@theme/Layout';
-import Link from '@docusaurus/Link';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
-import useBaseUrl from '@docusaurus/useBaseUrl';
-import styles from './styles.module.css';
+import React from "react";
```
Comment (hybridx) [inline packages/doc-spa/src/pages/index.js:1]: Didn't we have single quote as default? @akhilmhdh

**Review by hybridx — APPROVED** [review]
LGTM

File: packages/doc-spa/docs/getting-started/overview.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+The applications deployed in One Platform are,
+
+1. Open to all, by choice
+2. Hosted apps under single domain i.e one.redhat.com/yourapp
+3. Easy to search (including app contents) using common search service
+4. Have access to Core microservices (we will discuss them in detail in the next part)
+   - Authentication - Red Hat Single Sign-On enabled. (Subject to IT regulations)
+   - Authorization - Rover integration to view & grant users access.
+   - Feedback - Collect feedback in 3 clicks, Generate ticket (Jira, GitLab) for improvements,
```
Comment (deshmukhmayur) [inline packages/doc-spa/docs/getting-started/overview.md:34]:
```suggestion
   - Feedback - Collect feedback in 3 clicks, Generate ticket (Jira, GitLab) for improvements
```

File: packages/doc-spa/docs/services/apps-service/faqs.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+> Who are the App Service APIs for?
+
+The App Service APIs are for any developer who wants to use the metadata of an app they have registered on the One Platform.
+
+> Can anyone create an App? How do I create a new App?
+
+Yes. Anyone can create an app as long as you have an internal Red Hat account (i.e. as long as you are a Red Hat employee).
+
+To create a new App, it is recommended to use the Developer Console as it provides a graphical interface for quickly and easily creating an app/project on One Platform.
```
Comment (deshmukhmayur) [inline packages/doc-spa/docs/services/apps-service/faqs.md:18]:
```suggestion
To create a new App, it is recommended to use the [Developer Console](https://one.redhat.com/console) as it provides a graphical interface for quickly and easily creating an app/project on One Platform.
```

File: packages/doc-spa/docs/services/apps-service/onboarding-guide.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+title: Guides
+slug: /apps-service/guides
+sidebar_label: Guides
+---
+
+Any developer can use the Apps Service to view the metadata for their apps, and to configure the One Platform Service integrations.
+
+To get started with the Apps Service, follow the steps below:
+
+1. Register your app on the One Platform Developer Console
```
Comment (deshmukhmayur) [inline packages/doc-spa/docs/services/apps-service/onboarding-guide.md:12]:
```suggestion
1. Register your app on the [One Platform Developer Console](https://one.redhat.com/console)
```

File: packages/doc-spa/docs/services/apps-service/api-reference.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+id: api-ref
+title: API Reference
+slug: /apps-service/api-ref
+sidebar_label: API Reference
+---
+
+# API Reference
+
+You can test drive the available APIs on the [QA testing playground](https://api.qa.one.redhat.com/graphql).
```
Comment (deshmukhmayur) [inline packages/doc-spa/docs/services/apps-service/api-reference.md:10]:
```suggestion
You can test drive the available APIs on the [QA testing playground](https://qa.one.redhat.com/api/graphql).
```

File: packages/doc-spa/docusaurus.config.js [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+        },
+      }),
+    ],
+  ],
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      navbar: {
+        logo: {
+          alt: "My Site Logo",
```
Comment (deshmukhmayur) [inline packages/doc-spa/docusaurus.config.js:59]:
```suggestion
          alt: "One Platform Documentation",
```

File: packages/doc-spa/src/pages/markdown-page.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (deshmukhmayur) [inline packages/doc-spa/src/pages/markdown-page.md:1]: Is this page required? This looks like a placeholder from the docusaurus generator.

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]

File: packages/doc-spa/src/pages/index.js (hunk 1 above)
Comment (akhilmhdh) [reply packages/doc-spa/src/pages/index.js:1]: This is why we need a centralized eslint config. To make the style uniform and not depend on editor. I actually tried everything from vscode setting to editorconfig. Else need to modify editor config

File: packages/doc-spa/docs/contributing/code-of-conduct.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at one-portal@redhat.com. All
```
[the remark this answers is not in the document]
Comment (akhilmhdh) [reply packages/doc-spa/docs/contributing/code-of-conduct.md]: Done

File: packages/doc-spa/docs/deployment-guidelines/spa-deployment-guideline.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
+---
+id: spa-deployment-guideline
+title: OP SPA Deployment Guideline
+slug: /deployments/spa
+sidebar_label: SPA Guideline
+---
+
+SPAs in One Platform are deploying using a Jenkins ecosystem.
```
[the remark this answers is not in the document]
Comment (akhilmhdh) [reply packages/doc-spa/docs/deployment-guidelines/spa-deployment-guideline.md]: Done

File: packages/doc-spa/docs/getting-started/overview.md (hunk 2 above)
Comment (akhilmhdh) [reply packages/doc-spa/docs/getting-started/overview.md]: Done

File: packages/doc-spa/docs/services/apps-service/faqs.md (hunk 3 above)
Comment (akhilmhdh) [reply packages/doc-spa/docs/services/apps-service/faqs.md]: Done

File: packages/doc-spa/docs/services/apps-service/onboarding-guide.md (hunk 4 above)
Comment (akhilmhdh) [reply packages/doc-spa/docs/services/apps-service/onboarding-guide.md]: Done

File: packages/doc-spa/docs/services/apps-service/api-reference.md (hunk 5 above)
Comment (akhilmhdh) [reply packages/doc-spa/docs/services/apps-service/api-reference.md]: Done

File: packages/doc-spa/docusaurus.config.js (hunk 6 above)
Comment (akhilmhdh) [reply packages/doc-spa/docusaurus.config.js]: Done

**Review by deshmukhmayur — APPROVED** [review]

**Review by riginoommen — APPROVED** [review]

**Pull request merged**